hack/generate_local_env.py

```diff
@@ -33,7 +33,7 @@ def get_env_file(outdir, file_format='txt'):
         for row in reader:
             if row['image_var'] in ['VMWARE_IMAGE', 'CONVERSION_IMAGE', 'KUBEVIRT_VIRTIO_IMAGE']:
                 image = f"{row['name']}@sha256:{row['digest']}"
-                env = 'VIRTIOWIN_CONTAINER' if row['image_var'] == 'KUBEVIRT_VIRTIO_IMAGE' else row['image_var']
+                env = 'VIRTIOWIN_CONTAINER' if row['image_var'] == 'KUBEVIRT_VIRTIO_IMAGE' else row['image_var'].replace("_IMAGE", "_CONTAINER")
```

Comment:
why not storing them in `deploy/images.csv` with the proper suffix at first place?

Reply:
This is for running locally. This is a development issue. changing the tool rather than the product.